int-set/src/lib.rs

```diff
@@ -80,7 +82,21 @@ impl<T: Into<u32> + Copy> IntSet<T> {
         }
     }
 
-    // TODO(garrretrieger): add remove_range.
+    // Removes all values in iter from the set.
+    pub fn remove_all<U: IntoIterator<Item = T>>(&mut self, iter: U) {
```

Comment:
Do you anticipate this being useful as public API? I would imagine us eventually wanting various set operations (union, difference etc) which would overlap with this in specific cases, but it's not clear (as in, I actually don't know) to me how often we would be working with different sorts of iterators?

Reply:
I believe so, it provides an optimized alternative to just calling .remove() multiple times so useful any time you're doing a sequence of removals that aren't listed in another IntSet. Also since we need to implement this to support extend() on exclusive sets we might as well make it available in the public api.